app/src/main/res/drawable/left_arrow.xml

```diff
@@ -0,0 +1,26 @@
+<!--
```

Comment:
Is this something you have drawn yourself?

Reply:
No, it from android studio default vector library.